.github/workflows/wheels-test.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
           export MATRICES="
             pull-request:
               # amd64
-              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu18.04', gpu: 'v100', driver: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu18.04', gpu: 'v100', driver: 'earliest' }
               - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', gpu: 'v100', driver: 'latest' }
               # arm64
               - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
```

Comment:
I don't think we've ever seen wheel+arm be the only failing job, or at least not since the initial development of wheels. In theory there are potential glibc problems. In practice we've never observed them. I'm tempted to drop this because they're 10 minute jobs taking up our most valuable resource (arm GPU nodes).

Reply:
> taking up our most valuable resource (arm GPU nodes).

I'm not sure how true this is from recent observation. I think the sheer number of non-ARM jobs has led to long queue times for amd64 nodes even when no queue exists for arm64. I often see arm64 jobs finish _before_ amd64 due to the shorter queues. At one point, yes, arm64 had a longer queue, but I haven't seen that lately.

Reply:
I'm not sure if I understand the original comment though. Are you proposing to remove ARM wheel jobs from the PR matrix entirely? I think it's good to have at least one ARM job for wheels.

I would propose to leave an ARM + wheel job in the matrix for now. If we see ARM queue times increasing due to a change in the ratio of amd64 to ARM jobs, then we can reconsider.